cmd/exec_unix.go

```diff
@@ -23,3 +27,31 @@ func exec(ctx context.Context, command string, args []string, env []string) erro
 	// Only return if the execution fails.
 	return errors.Wrap(syscall.Exec(argv0, argv, env), "error executing command")
 }
+
+func getAWSEnvVars(assumeRoleOutput *sts.AssumeRoleWithWebIdentityOutput, awsOIDCConfig *aws_config_client.AWSOIDCConfiguration) []string {
+
+	// Load config profile values if those environment variables don't exist (lowest precedence)
+	envVars := []string{}
+	_, present := os.LookupEnv("AWS_REGION")
+	if (awsOIDCConfig.Region != "") && !present {
+		envVars = append(envVars, fmt.Sprintf("AWS_REGION=%s", awsOIDCConfig.Region))
+	}
```

Comment:
```suggestion
	_, present := os.LookupEnv("AWS_DEFAULT_REGION")
	if !present && (awsOIDCConfig.Region != nil) {
		envVars = append(envVars, fmt.Sprintf("AWS_DEFAULT_REGION=%s", *awsOIDCConfig.Region))
	}
```
- Seems like AWS is [converging](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) on AWS_DEFAULT_REGION. Feels like we should too. 
- Instead of using the empty string to denote "absence" we can use the *string and do a nil check. 
- Reordering of the conditional check is purely sytnax preference (simpler condition first), feel free to revert if you prefer the other way.